rllib/models/tf/recurrent_net.py

```diff
@@ -178,7 +180,9 @@ def __init__(self, obs_space: gym.spaces.Space,
         self._rnn_model = tf.keras.Model(
             inputs=[input_layer, seq_in, state_in_h, state_in_c],
             outputs=[logits, values, state_h, state_c])
-        self._rnn_model.summary()
+        # Print out model summary in debug mode.
+        if logger.isEnabledFor(logging.INFO):
```

Comment:
nit: either the comment or the logging level is incorrect?

Reply:
+1
changed